### ISSUE TITLE
TYPO3 Exception: Unknown Namespace: libconnect

### ISSUE THREAD

**Issue by johlton** [subhh/libconnect#14]
With Typo3 8.7.32 in composer mode, I get the uncaught exception error:

> Fluid parse error in template partial_Dbis/TopList_629d4779a49990829c186d847f9648ba6dab7019, line 3 at character 86. Error: Unknown Namespace: libconnect (error code 0). Template source chunk: <libconnect:strreplace search=" & " replace=" &amp;">
> 
> TYPO3Fluid\Fluid\Core\Parser\UnknownNamespaceException thrown in file
> /home/fab/_srv/gitlab/typo3-tub/vendor/typo3fluid/fluid/src/Core/Parser/TemplateParser.php in line 158.

I installed it with Extension Manager, though (with the workaround via `vendor/typo3/autoload-include.php`, since there is no composer package available yet). 

I'm trying to include it into my pretty basic FLUID template with the following TypoScript:

```
plugin.tx_libconnect {
     ezbbibid = XYZ
     dbisbibid = xyz

     view {
         templateRootPath = EXT:libconnect/Resources/Private/Templates/
         partialRootPath = EXT:libconnect/Resources/Private/Partials/
         layoutRootPath = EXT:libconnect/Resources/Private/Layouts/
     }

     settings {
         ezbNoCSS = 1
         dbisNoCSS = 1
     }
}
```

What is going wrong here? This should work out of the box with Typo3 8, correct? Am I missing something, perhaps also in my (fresh) Typo3 installation?

**This only affects the DBIS view, the EZB view is fine.**

Thanks in advance!

**Comment by percolatorWitt** [comment]
Hi,

we use TYPO3 8.7 too on our website. I will check the problem.

**Comment by percolatorWitt** [comment]
It´s fixed with 7.0.5.
